src/main/kotlin/org/jetbrains/plugins/bsp/magicmetamodel/impl/workspacemodel/impl/updaters/transformers/ModuleDetailsToGoModuleTransformer.kt

```diff
@@ -39,9 +39,10 @@ internal class ModuleDetailsToGoModuleTransformer(
         )
       }),
       resourceRoots = resourcesItemToGoResourceRootTransformer.transform(inputEntity.resources),
-      importPath = goBuildInfo.importPath,
+      importPath = goBuildInfo?.importPath ?: "",
       root = URI.create(inputEntity.target.baseDirectory).toPath(),
-      goDependencies = toGoDependencies(inputEntity)
+      goDependencies = toGoDependencies(inputEntity),
+      goLibraries = inputEntity.goLibraries ?: emptyList(),
```

Comment:
Jeśli z input entity usuniemy `goLibraries`, to `projectDetails.libraries` w połączeniu z `inputEntity.libraryDependencies` da nam ten sam rezultat i powinno rozwiązać problem dodawania, tylko libraries od których jesteśmy zależni

Reply:
Tak dodatkowo, to w sumie w tym `libraryDependencies` jest chyba więcej niż powinno być, to znaczy są tam wszystkie targety z tego `org_golang_x_net` zamiast tylko `ipv4` ale to powiem abramsowi jako potencjalny bug - tak czy inaczej, jeśli tam jest bug, to robiąc to w ten sposób będziemy mieli tylko jedno miejsce do naprawiania zamiast dwóch